src/task/seventh/Pair.java

```diff
@@ -0,0 +1,51 @@
+package task.seventh;
+
+import java.util.Objects;
+
+class Pair <T, L>{
+    private T first;
+    private L second;
+
+    private Pair(T first, L second) {
+        this.first = first;
+        this.second = second;
+    }
+
+    public T getFirst() {
+        return first;
+    }
+
+    public L getSecond() {
+        return second;
+    }
+
+    public static void main(String[] args) {
+        Pair<Integer, String> pair = Pair.of(1, "hello");
+        Integer i = pair.getFirst(); // 1
+        String s = pair.getSecond(); // "hello"
+
+        Pair<Integer, String> pair2 = Pair.of(1, "hello");
+        boolean mustBeTrue = pair.equals(pair2); // true!
+        boolean mustAlsoBeTrue = pair.hashCode() == pair2.hashCode(); // true!
+        System.out.println(mustAlsoBeTrue);
+        System.out.println(mustBeTrue);
+    }
+
+    public static <T, K> Pair<T, K> of(T first, K second) {
```

Comment:
private static <T, K> Pair <T, K> of(K key, V value) ->
private static <T, K> Pair of(K key, V value)